CHANGELOG/CHANGELOG-2.6.md

```diff
@@ -0,0 +1,23 @@
+# Release notes for v2.6.0
```

Comment:
Since there is just one bug fix, should we consider doing a patch release instead of a minor release?